sdk/app_cpu1/common/sys/scheduler.h

```diff
@@ -9,10 +9,15 @@
 #include "usr/user_config.h"
 
 // Utility defines for time conversions
-#define USEC_IN_SEC       (1000000)
+#define USEC_IN_SEC       (1000000.0)
```

Comment:
Why change to `double` type constant?

I get that the code uses `double` type to record time... but, seems like `USEC_IN_SEC` should be an integer million value. I think the macros below would still work fine with this? or, users can cast the input to the macros and the compiler should up-cast the `USEC_IN_SEC` value..?